test/index.js

```diff
@@ -0,0 +1,32 @@
+import {assert} from 'chai'
```

Comment:
do you mind using jest? we are migrating everything in jss to jest

Reply:
Gotcha - yes I'll switch it to jest